### ISSUE TITLE
Angular 18 Support

### ISSUE THREAD

**Issue by s-torrisi** [thekip/angular-cc-library#124]
At the moment the library does not support angular 18

Is there anything I can do to help update it or is there a planned release date?

**Comment by wen9218** [comment]
@thekip ,  I am waiting for new release of matching angular 18
Thanks

**Comment by born2net** [comment]
yes please, need to move to ngv18 and waiting on this lib <3 TX!

**Comment by cwithhart** [comment]
Any update on this? 18 has been out for quite sometime.

**Comment by cwithhart** [comment]
Can someone help me out. I tried updating the library to 18 but getting an issue when running ng test. It looks like the require() for non es modules isnt working and im not exactly sure the answer.
https://github.com/cwithhart/angular-cc-library/tree/angular-18-update

**Comment by s-torrisi** [comment]
@cwithhart can you add me as maintainer to your fork? I created a pull request that would fix the problem with the tests (It is mainly related to an old yarn version and to standalone components)

**Comment by born2net** [comment]
that would be great if more people will work on the project, tx

**Comment by s-torrisi** [comment]
@thekip, can you add me as maintainer to this project too? I'd be glad to help maintain this project also in the future

**Comment by thekip** [comment]
guys, if you want to participate i would be glad to accept PR's 

I think updating to the Angular 18 should be as easy as just a peer dep version bump.

**Comment by s-torrisi** [comment]
@thekip the publishing of the app has failed, have you had a chance to take a look at it?

**Comment by s-torrisi** [comment]
@timofei-iatsenko any news?

**Comment by timofei-iatsenko** [comment]
Sorry, was busy on the job. Will try to bring the pipeline to live now

**Comment by timofei-iatsenko** [comment]
https://www.npmjs.com/package/angular-cc-library/v/3.4.0
